Android/app/src/main/java/com/example/boardgamesocial/LoginFragment.java

```diff
@@ -52,4 +64,40 @@ public void onDestroyView() {
         binding = null;
     }
 
+    private void processUserLogin() {
+
+        String username = Objects.nonNull(etUsername.getText()) ? etUsername.getText().toString() : null;
+        String password = Objects.nonNull(etPassword.getText()) ? etPassword.getText().toString() : null;
+
+        Log.i("Login", "Username: "+ username +", Password: "+ password);
+
+        retrofitClient.loginCall(new User(username, password)
+        ).enqueue(new Callback<Token>() {
+            @Override
+            public void onResponse(Call<Token> call, Response<Token> response) {
+                if (response.isSuccessful()) {
+                    assert response.body() != null;
+                    token = response.body();
+                    Log.i("Token", token.toString());
+                    Toast.makeText(getContext(), "Login Successful", Toast.LENGTH_LONG).show();
+                    Intent goToHomePostActivity = HomePostActivity.getIntent(getContext());
```

Comment:
remember that we changed to a two activity setup
```suggestion
                    Intent goToHomePostActivity = MainAppActivity.getIntent(getActivity().getApplicationContext());
                    startActivity(goToHomePostActivity);
```